hail/build.gradle

```diff
@@ -42,12 +42,12 @@ tasks.withType(JavaCompile) {
 project.ext {
     breezeVersion = "1.1"
 
-    sparkVersion = System.getProperty("spark.version", "3.3.0")
+    sparkVersion = System.getProperty("spark.version", "3.3.2")
 
-    if (sparkVersion != "3.3.0") {
-        project.logger.lifecycle("WARNING: Hail primarily tested with Spark 3.3.0, use other versions at your own risk.")
+    if (sparkVersion != "3.3.2") {
+        project.logger.lifecycle("WARNING: Hail primarily tested with Spark 3.3.2, use other versions at your own risk.")
```

Comment:
This isn't exactly accurate though, the pinned requirements say we're using `pyspark==3.3.3`. I think we should either omit the patch version or use what is in our `pinned-requirements.txt`.

Reply:
Good catch. Hmm. I really want us to test with the version of Spark that is used in Dataproc but I don't want to put a pin in the public requirements. Is it possible to constrain the pinned pyspark without the addition of a new requirements file or should I add a `hail/python/constraints-requirements.txt` or w/e?

Reply:
Ya I think if you wanted to do that and keep `requirements.txt` open then you would have to add another requirements file with `pyspark==3.3.2`. Or, if what you're really after is just keeping the wheel open then you could tweak the code already in `setup.py` to unconditionally open up the `pyspark` requirement and keep `requirements.txt` pinned.